Source/Atomic/UI/UIEditField.cpp

```diff
@@ -247,6 +260,22 @@ bool UIEditField::OnEvent(const tb::TBWidgetEvent &ev)
                 styleEdit->selection.SelectAll();
             }
         }
+
+#if defined(ANDROID) || defined( __ANDROID__) || defined(IOS)
+
+        // triple click to get the onscreen keyboard, in case it is auto-focused
+        else if ( ev.count == 3 ) 
+        {
```

Comment:
Or as I like to call it,  the "angry click"... I have no idea if it is a standard, but I suspect that no one has been entering data on android+atomic+turbobadger, and this makes it at least possible and not too annoying.  There are many more issues with TB on android to clean up.